elements/rh-accordion/rh-accordion-header.css

```diff
@@ -15,6 +15,7 @@
   --pf-c-accordion__toggle--active-text--Color: #151515;
   --pf-c-accordion__toggle--expanded-text--Color: #151515;
   --pf-c-accordion__toggle--expanded-icon--Rotate: -180deg;
+  --pf-c-accordion__toggle--before--Top: -1px;
```

Comment:
ideally, we'd  move the pf styles to pfe-accordion's scss  files  and redo all these with tokens or hardcoded values

Reply:
Agreed, this was after playing around with closing the gap between the panel and the header for the after element but this should be done in pfe by default because it'll need to happen over there as well.

Reply:
Ok. For now, I suggest changing this PR so that it assumes there are no pf vars in baseaccordion, which will probably mean reimplementing all that values here

Reply:
Wait, are you saying we rename all the variables in BaseAccordionHeader.scss in pfe so that it doesn't have the --pf-c beginning?  Like just doing `--accordion__toggle--before--Top`?

Reply:
or do you more mean that all of these that are set not using tokens but instead using hardcoded values like `--pf-c-accordion__toggle--active-text--Color: #151515;` should be `--pf-c-accordion__toggle--active-text--Color: var(--rh-color-text-primary-on-light, #151515);